package.json

```diff
@@ -2,159 +2,93 @@
   "name": "@ipld/dag-json",
   "version": "0.0.0-dev",
   "description": "JS implementation of DAG-JSON",
-  "main": "index.js",
-  "types": "./types/index.d.ts",
-  "type": "module",
-  "scripts": {
-    "lint": "standard *.js test/*.js",
-    "build": "npm run build:js && npm run build:types",
-    "build:js": "ipjs build --tests --main && npm run build:copy",
-    "build:copy": "cp -a tsconfig.json index.js dist/ && mkdir -p dist/test && cp test/*.js dist/test/",
-    "build:types": "npm run build:copy && cd dist && tsc --build",
-    "test:cjs": "npm run build:js && mocha dist/cjs/node-test/test-*.js && npm run test:cjs:browser",
-    "test:esm": "npm run build:js && mocha dist/esm/node-test/test-*.js && npm run test:esm:browser",
-    "test:node": "c8 --check-coverage --branches 100 --functions 100 --lines 100 mocha test/test-*.js",
-    "test:cjs:browser": "polendina --page --worker --serviceworker --cleanup dist/cjs/browser-test/test-*.js",
-    "test:esm:browser": "polendina --page --worker --serviceworker --cleanup dist/esm/browser-test/test-*.js",
-    "test:ts": "npm run build:types && npm run test --prefix test/ts-use",
-    "test": "npm run lint && npm run test:node && npm run test:esm && npm run test:ts",
-    "test:ci": "npm run lint && npm run test:node && npm run test:esm && npm run test:cjs && npm run test:ts",
-    "coverage": "c8 --reporter=html mocha test/test-*.js && npm_config_yes=true npx st -d coverage -p 8080"
+  "author": "Rod <rod@vagg.org> (http://r.va.gg/)",
+  "license": "Apache-2.0 OR MIT",
+  "homepage": "https://github.com/ipld/js-dag-json#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ipld/js-dag-json.git"
+  },
+  "bugs": {
+    "url": "https://github.com/ipld/js-dag-json/issues"
   },
   "keywords": [
     "ipfs",
     "ipld",
     "multiformats"
   ],
-  "author": "Rod <rod@vagg.org> (http://r.va.gg/)",
-  "license": "(Apache-2.0 AND MIT)",
+  "engines": {
```

Comment:
these don't happen to be kept updated by unified-ci do they? it concerns me that we're introducing more fields that need manual attention to keep current

Reply:
I had a similar concern - they are [defined here](https://github.com/ipfs/aegir/blob/77e43ce61be5e1394a2b8306f122cd9303063499/src/check-project/utils.js#L249-L252) and kept up to date by the `aegir check-project` command.

Long(ish) term I was hoping for a `aegir check-project -y` type thing we could run as part of unified CI to automate that too, I wanted to get a view on how much custom config we have across the estate first though - turns out not much so that sounds doable at some point.  I think the biggest thing is the divergence in readme styles but I'm sure there's a middle ground somewhere.